utils/symbol.c

```diff
@@ -201,7 +201,7 @@ static void unload_symtab(struct symtab *symtab)
 }
 
 static int load_symbol(struct symtab *symtab, unsigned long prev_sym_value,
-		       unsigned long offset, unsigned long flags,
+		       unsigned long long offset, unsigned long flags,
```

Comment:
I don't get it.  Could you please provide a concrete example?

Reply:
I'll update detail of this.

Reply:
@namhyung I described it at commit message. please refer here : b9e7fcdd6f4f1043f07d9b57979558a40910eb71

Reply:
Thanks for the explanation